COPYRIGHTS

```diff
@@ -0,0 +1,26 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: test-definitions
+Upstream-Contact: Linaro validation mailing list <linaro-validation@lists.linaro.org>
+Source: https://github.com/Linaro/test-definitions
+
+Files: *
+Copyright: 2012-2019 Linaro Limited
+           2019 Daniel Wagner <wagi@monom.org>
+           2019 Daniel Wagner <daniel.wagner@siemens.com>
+           2019 Patryk Mungai <patryk.mungai-ndungu.kx@renesas.com>
+           2018 Karsten Tausche <karsten@fairphone.com>
+           2018 Oleksandr Terentiev <oterenti@cisco.com>
+           2017 Lei Yang <Lei.Yang@windriver.com>
+License: GPL-2.0-only
```

Comment:
With this many copyright holders, I would expect one paragraph for each. But this surely works.

Reply:
I checked all contributors from git log. Is this a correct setup? I can surely split that into separate entries with paths. Some are easy as the contribution was adding new tests. Some cases are more tricky as they were changing a few existing files. Any advice on that?

Reply:
nah I think this should be ok.